File: src/main/java/ar/edu/utn/frba/dds/domain/services_api/georef/entities/Provincia.java
package ar.edu.utn.frba.dds.domain.services_api.georef.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Provincia {
    @Id
    public int id;
    @Column(name = "nombre")
    public String nombre;

}

File: src/main/ts/ar/edu/utn/frba/dds/domain/services_api/georef/entities/Provincia.ts
import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';

@Entity()
export class Provincia {
    @PrimaryGeneratedColumn()
    id: number;

    @Column({name: "provincia"})
    nombre: string;

    constructor (nombre: string) {
        this.nombre = nombre;
    }
}
